src/components/SafeListSidebar/style.ts

```diff
@@ -3,17 +3,18 @@ import { makeStyles } from '@material-ui/core/styles'
 import { headerHeight, md, mediumFontSize, screenSm, secondaryText, xs } from 'src/theme/variables'
 
 const sidebarWidth = '400px'
-const sidebarMarginLeft = '12px'
-const sidebarMarginTop = '12px'
-const sidebarMarginBottom = '12px'
-const sidebarBorderRadius = '8px'
+const sidebarMarginLeft = '0px'
+const sidebarMarginTop = '0px'
+const sidebarMarginBottom = '0px'
+const sidebarBorderRadius = '0px'
```

Comment:
do we still need to specify it if it's 0?

Reply:
I kept it because it does some height calc magic depending on the these params. Might be useful if the design changes again. I could remove the calc though.